plenum/server/node.py

```diff
@@ -2102,21 +2125,27 @@ def lost_master_primary(self):
         self._schedule(self.propose_view_change,
                        self.config.ToleratePrimaryDisconnection)
 
-    def startViewChange(self, proposedViewNo: int):
+    def startViewChange(self, proposed_view_no: int):
         """
         Trigger the view change process.
 
-        :param proposedViewNo: the new view number after view change.
+        :param proposed_view_no: the new view number after view change.
         """
         # TODO: consider moving this to pool manager
         # TODO: view change is a special case, which can have different
         # implementations - we need to make this logic pluggable
 
+        for view_no in tuple(self._next_view_indications.keys()):
+            if view_no <= proposed_view_no:
```

Comment:
ok, but what is the point, do you see more iterations being done the way it is now?